addons/painkillers/ACE_Medical_Actions.hpp

```diff
@@ -0,0 +1,20 @@
+class ACE_Medical_Actions {
+    class Basic {
+        class Morphine;
+        class PainKillers: Morphine {
+            displayName = CSTRING(DisplayName);
+            displayNameProgress = CSTRING(painkillerProgress);
+            allowedSelections[] = {"head"};
+            items[] = {"TAC_painkillers"};
```

Comment:
Hmm, `tac_painkillers` would be better. But `QUOTE(painkillers)` with component named `tac_medical` would be even better I think.